package.json

```diff
@@ -1281,6 +1292,49 @@
           "type": "integer",
           "default": 1,
           "description": "Number of header rows to assume. Set to 0 to disable.`"
+        },
+        "latex-workshop.formattedPaste.image.method": {
```

Comment:
I am strongly concerned about moving the image files. I suppose we can simplify the workflow. If it is an image file, reference its absolute path. If it's a clipboard image, save in default folders as below. My 2c.

Reply:
Seems reasonable. I'll just hardcode that to `"leave"`. The one extra bit I'll leave is if text is selected when pasting — then the image will just be copied over. The only added complexity due to this is the following three lines.

```ts
import { promisify } from 'util'
const fsCopy = promisify(fs.copyFile)
...
fsCopy(oldPath, imagePath)
```
Done in e59e6ed02fa660a75e6f300790018c7c5bb65b66